packages/explorer/README.md

```diff
@@ -1,85 +1,53 @@
 # World Explorer
 
 World Explorer is a GUI tool designed for visually exploring and manipulating the state of worlds.
+For the full information about it, [see the docs](http://mud.dev/world-explorer).
 
-## Getting started
+## Installation
 
-1. **Install the package**
+Starting with MUD 2.2, the MUD templates come with the World Explorer installed.
+These instructions are how to use World Explorer on earlier versions of MUD.
 
-   ```sh
-   pnpm add @latticexyz/explorer
-   ```
-
-2. **Start a local development chain**
-
-   Ensure you have a local development chain running.
+The easiest way to get World Explorer for earlier MUD versions is to create a project with the new template.
 
-3. **Run the World Explorer**
+1. Create a project with the new template, which has World Explorer.
+   Then, delete the files you no longer need.
 
-   ```sh
-   npx @latticexyz/explorer --worldAddress <YOUR_WORLD_ADDRESS>
+   ```sh copy
+   pnpm create mud@main explorer --template vanilla
+   cd explorer
+   rm -rf packages/client packages/contracts/[st]*
    ```
 
-   Alternatively, if you have a worlds configuration file:
-
-   ```sh
-   npx @latticexyz/explorer --worldsConfigPath <PATH_TO_WORLDS_CONFIG>
+1. Edit `mprocs.yaml` to remove the definitions for `client`, `contracts`, and `anvil`.
+
+   ```yaml filename="mprocs.yaml" copy
+   procs:
+     indexer:
+       cwd: packages/contracts
+       shell: rimraf $SQLITE_FILENAME && pnpm sqlite-indexer
+       env:
+         RPC_HTTP_URL: "http://127.0.0.1:8545"
+         FOLLOW_BLOCK_TAG: "latest"
+         SQLITE_FILENAME: "indexer.db"
+     explorer:
+       cwd: packages/contracts
+       shell: pnpm explorer
    ```
 
-   Note: You can use `@latticexyz/store-indexer` for indexing your world's data.
-
-## CLI arguments
```

Comment:
I think this ought to be included in README as well if we are including installation steps.